templates/.eslintrc.json

```diff
@@ -22,13 +22,10 @@
   "extends": [
     "airbnb",
     "plugin:jest/recommended",
-    "plugin:prettier/recommended"
+    "plugin:prettier/recommended",
+    "prettier",
+    "prettier/react"
   ],
-  "plugins": [
-    "jest",
-    "prettier"
-  ],
-  "rules": {
-    "prettier/prettier": "error"
-  }
+  "plugins": ["jest", "prettier"],
```

Comment:
Same here, looks like `plugin:prettier/recommended` already takes care of this?

Reply:
And I guess we could even drop `"jest"` from the plugins list?

https://github.com/jest-community/eslint-plugin-jest/blob/master/src/index.ts#L42